protocols/v2/codec-sv2/Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "MIT"
 
 [dependencies]
 serde = { version = "1.0.89", default-features = false, optional = true }
+snow = "0.8"
```

Comment:
code- sv2 should depend on snow only when is compiled with the with noise flag. You can make it optional or you could reexport snow from noise-sv2 I think that the latter is a better option

Reply:
this dependency was added only to support the new `codec_sv2::error::Error::SnowError` variant and i was wondering if there was a better alternative than importing the package here. i like the idea of re-exporting from `noise-sv2`, and will change to do that